### ISSUE TITLE
[Cemetery Status] [A11y] Screen reader is not reading "main phone number"

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#20723]
## Description
When run through screen reader in ( Arlington, Vermont, United States) instead of reading "Main phone number link with phone number 518-581-9128, its reading only Link, 518-581-9128. 

### User story

**AS A** Veteran, their caregiver(s), family member(s), and survivor(s) using Facility Locator in VA.gov to find cemeteries
**I WANT** I want to be able to use screen readers to navigate and access information,
**SO THAT** I can easily find the resources I need.

## Acceptance criteria
- [ ] Main phone should include an `aria-describedby` parameter
- [ ] Screen reader should announce "main phone" before reading the phone number
- [ ] Link the aria-describedby attribute of the phone number to the ID of the descriptive text element.  Example:  <a href="tel:518-581-9128" aria-describedby="main-phone-label">518-581-9128</a>
- [ ] The aria-describedby="main-phone-label" attribute tells the screen reader to first read the content of the span with the ID "main-phone-label" (which is "Main phone") before reading the phone number.

- [ ] Does this product have an existing regression test plan?
	- [x] Yes, [link](https://dvagov.sharepoint.com/:x:/r/sites/SitewideContract/Shared%20Documents/Regression%20Tests/Facility%20Locator%20regression%20testing.xlsx?d=wd5e3dc3652f94a11bc13cfb011e8f77e&csf=1&web=1&e=xbG3iy&nav=MTVfezAwMDAwMDAwLTAwMDEtMDAwMC0wNDAwLTAwMDAwMDAwMDAwMH0):

**Comment by Shamiaalam** [comment]
@chriskim2311 when I run through screen reader in ( Arlington, Vermont, United States) instead of reading "Main phone number link with phone number 518-581-9128, its reading only Link, 518-581-9128.  So its an issue as its supposed to be read main. phone. number as well. @FranECross

**Comment by Shamiaalam** [comment]
@FranECross So basically I run through voiceover in iOS 15.3.1 and it was reading Main phone number with phone link! Which is right. so it just might be a difference in VO versions and we can't do anything about that unfortunately. And I discussed with Laura, she got same result with main phone number. But it’s just different version of voice over thing that reading like that. VoiceOver (VO) has had known issues with aria-describedby in certain contexts, particularly when dynamically updating content. Sometimes I also noticed that VO read differently through different version. So coding is right.  it’s just voiceover that acting weird with different version. So we can close this ticket if  @chriskim2311 ok with that!

**Comment by FranECross** [comment]
@Shamiaalam Thank you for looking at this again (and Laura, too!). Yes, this is fine to close, since it's not a coding issue. @chriskim2311 Closing.

**Comment by chriskim2311** [comment]
@Shamiaalam Do you have the difference in VO versions that were causing the issues? Were they newer or older? We can pass on this information to the DST team to make changes to allow more devices to be able to use VO when interacting with DST components. Thanks!

**Comment by Shamiaalam** [comment]
@chriskim2311 yes, the newer version VO is reading properly with MAIN phone number when i run the test. Only older version is having issue! yes, that would be great because its confusing. in VO to read different from older and newer one.
